tests/test_config_consistency.py

```diff
@@ -71,3 +71,30 @@ def test_status_code_202(self):
         assert spans[0]["type"] == "web"
         assert spans[0]["meta"]["http.status_code"] == "202"
         assert spans[0]["error"] == 1
+
+
+# Tests for verifying default query string obfuscation behavior can be found in the Test_StandardTagsUrl test class
+@scenarios.tracing_config_nondefault_2
+@features.tracing_configuration_consistency
+class Test_Config_ObfuscationQueryStringRegexp_Empty:
+    """ Verify behavior when set to empty string """
+
+    def setup_query_string_obfuscation_empty(self):
+        self.r = weblog.get("/?application_key=123")
+
+    def test_query_string_obfuscation_empty(self):
+        interfaces.library.add_span_tag_validation(
+            self.r, tags={"http.url": r"^.*/\?application_key=123$"}, value_as_regular_expression=True,
+        )
+
+
+@scenarios.tracing_config_nondefault
```

Comment:
Fwiw, we have several tests checking that the default regexp works and is up to data, at `tests/test_standard_tags.py`. But +1 to test when configured to a non-default value.

Reply:
Hi @smola , yes I tried to expose that information with my comment on line 76 😄 . Is it clear enough or are you suggesting I change it to the file name?